packages/core-flows/src/definition/cart/workflows/add-to-cart.ts

```diff
@@ -22,6 +23,19 @@ import { prepareConfirmInventoryInput } from "../utils/prepare-confirm-inventory
 import { prepareLineItemData } from "../utils/prepare-line-item-data"
 import { refreshPaymentCollectionForCartStep } from "./refresh-payment-collection"
 
+const cartFields = [
```

Comment:
This is a lot of data, do we really need everything? I'm afraid this might have a toll on performance.

Reply:
These actions are performed on a cart context, depending on what the rules are, they could be any of these. We can optimize this by whitelisting what we allow as "rules", but that can come later.